spec/support/shared_examples/features/admin_content_nodes_spec.rb

```diff
@@ -0,0 +1,171 @@
+require 'rails_helper'
+
+RSpec.shared_examples :admin_content_node do
+  let(:node_button) { node_name.to_s.humanize }
+  let!(:subject_node) { FactoryGirl.create node_name, author: account }
```

Comment:
let's separate 'let' and 'let!' with empty lines as different logic blocks for better formatting